docker/grpc-root.dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:20.04
```

Comment:
This image only needs to be rebuilt infrequently as it takes ages (building gRPC and protobuf)